controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java

```diff
@@ -320,24 +346,49 @@ else if (timedOut(id, deployment.get(), endpointTimeout)) {
                 return Optional.of(error);
             }
         }
+        controller.jobController().locked(id, lockedRun -> lockedRun.withSummary(summary));
 
         if (timedOut(id, deployment.get(), installationTimeout)) {
+            logger.log(nodeList.asList().stream()
+                               .flatMap(node -> nodeDetails(node, true))
+                               .collect(toList()));
             logger.log(INFO, "Installation failed to complete within " + installationTimeout.toMinutes() + " minutes!");
             return Optional.of(installationFailed);
         }
 
-        logger.log("Installation not yet complete.");
```

Comment:
(I liked this message in the original, as it made it crystal clear I need to wait longer. But it could be this is superfluous with the other changes...)

Reply:
Should be covered by the High-level status, indeed.